releasenotes/notes/batched-shots-gpu-1d8449a6df40d62b.yaml

```diff
@@ -1,18 +0,0 @@
----
-features:
-  - |
-    Adds a batched shot simulation optimization for GPU simulations. This
-    optional will use available memory on 1 or more GPUs to run multiple
-    simulation shots in parallel for greatly improved performance on
-    multi-shot simulations with noise models and/or intermediate measurements.
-
-    This option is enabled by default when using ``device="GPU"`` for the
-    :class:`~qiskit.providers.aer.AerSimulator` for ``"statevector"`` and
-    ``"density_matrix"`` simulation methods and can be disabled by setting the
-    ``batched_shots_gpu=False`` in the simulator options.
-
-    This optimization is most benefical for small to medium numbers of qubits
-    where there is sufficient GPU memory to run multiple simulations in
-    parallel. The maximum number of activate circuit qubits for enabling this
-    optimization can be configued using the ``batch_shots_gpu_max_qubits``
-    simulator option. The default value of this option is 16.
```

Comment:
How come a bunch of release notes are deleted and copied into one master note?

Reply:
Just personal preference, helps me to organize them when reviewing all the notes for the release.